easyrest/views/auth_controler.py

```diff
@@ -59,6 +61,65 @@ def check_token(request):
     }
     return wrap(data)
 
+@view_config(route_name='login', renderer='json', request_method='PUT')
+def login_openid(request):
+    """Controler for recivind google id_token from react redirect. 
+    Expects:
+        PUT request:
+            {'id_token': token}
+    Return:
+        {
+            "data": [],
+            "status": True if user added good
+                      False if user added bad
+        }
+    """
+    req_json = request.json_body
+    id_token = req_json.get("id_token", False)
+    if not id_token:
+        raise HTTPNotFound("No id_token suplied")
+    
+    decoded_token = jwt.decode(id_token, 'NrbEQgu9ggA4VJqmdtynWgcs', verify=False, algorithms=['HS256'])
+
+    email = decoded_token.get('email', False)
+    new_pass = decoded_token.get('jti', False)
+    name = decoded_token.get('name', '').encode("utf-8")
+    # import codecs
+    print(name)
+    # print(name.decode("UTF-8").encode("UTF-8"))
+    img = decoded_token.get('picture', '')
+
+    if not email or not new_pass:
+        raise HTTPBadRequest("no email in token")
+
+    user = request.dbsession.query(User).filter_by(email=email).first()
+    # check if user exists    
+    if user is None:
+        form_data = {
+            "email": email,
+            "password": new_pass,
+            "name": decoded_token.get('name', ''),
```

Comment:
You done this above and create variable. You cat use it here

Reply:
Lol, didn't see that